### DISCUSSION TITLE
Expedition 8 read log issue.

### DISCUSSION THREAD

**Discussion by RedDeuces** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#33] [General]
At the very start of Expedition 8 I do not have an option to speak to the captain to read the logs. I saw in the readme that this was an issue previously. I'm not sure if something else has cropped up. I loaded up another expedition and then refreshed this one to see if it was a one time bug, but the 2nd attempt is still unable to progress.

**Comment by cwmonkey** [comment]
@RedDeuces try grabbing the new json from here: https://cwmonkey.github.io/nms-expeditions/

It should be the same as the one in this repo, but you may have a stale copy. After getting the new json file I would also load a game that has a freighter that is NOT a pirate freighter. After verifying the freighter is not a pirate freighter, save your game, exit, then try starting expedition 8.

**Comment by RedDeuces** [comment]
@cwmonkey That seemed to resolve it. One thing I noticed was that in the origianl attempts the Captain wasn't a captain at all. It was a Raid leader or some other named character. I'm guessing it was the pirate NPCs that were trying to load even though it was a Gek civilian ship. Thanks so much for the fast response. You have been so helpful!

**Reply by cwmonkey** [reply]
Yeah the pirate freighter thing was actually a bug that existed at the time of the expedition, and they just never fixed it. There are a few of those types of bugs, and a few new ones that came up after the expeditions were over (like the death reset mechanic not working in Leviathan). Unfortunately this isn't a mod so we can't fix their bugs, best we can do is figure out how to work around them.
